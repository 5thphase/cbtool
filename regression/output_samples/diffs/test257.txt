|name |role |size |cloud_ip |host_name |vmc_pool |netname |ai |aidrs |abort |uuid
|vm_XX |type |sizeXX |undefined |unavailable |POOL |private |none |none |False |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|vm_XX |foorole |from_vm_template|undefined |unavailable |unavailable |private |none |none |False |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|vm_XX |type |sizeXX |undefined |unavailable |POOL |private |none |none |False |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
