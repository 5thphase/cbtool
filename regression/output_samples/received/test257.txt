The following FAILED VMs are in this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool            |ai      |aidrs      |uuid
|vm_23         |norole              |default     |undefined       |unavailable                |SUT                 |none    |none       |0E77C138-75E6-5971-927F-197A1A7FA5E5
|vm_25         |willfail            |nano32      |undefined       |unavailable                |SUT                 |none    |none       |FD9A5271-C9BA-5838-88CA-002B066E6AD2
|vm_24         |willfail            |nano32      |undefined       |unavailable                |SUT                 |none    |none       |FC9700C6-2064-520F-A0B8-897179046327

