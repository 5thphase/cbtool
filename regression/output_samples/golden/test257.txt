The following FAILED VMs are in this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool      |netname    |ai      |aidrs      |uuid
|vm_23         |norole              |from_vm_template|undefined       |unavailable                |unavailable   |private    |none    |none       |F2778E8A-E566-56ED-999F-D3A382EB4A37
|vm_24         |willfail            |nano32      |undefined       |unavailable                |SUT           |private    |none    |none       |760685B2-BC54-5851-B01E-FAAE51727F4A
|vm_25         |willfail            |nano32      |undefined       |unavailable                |SUT           |private    |none    |none       |44B9F311-528B-59B0-A435-B0F2D89DEBED

